explorer/client/src/components/JobRuns/TaskRuns.tsx

```diff
@@ -37,6 +37,9 @@ const styles = ({ spacing, palette }: Theme) =>
       alignItems: 'center',
       flexGrow: 1
     },
+    pendingConfirmations: {
```

Comment:
Maybe it will be better to merge pendingConfirmations and etherscan keys into a generic key name (since the two are identical now) and apply it to both?

Reply:
I think it's fine as is, separate with no fear of collision.